05a-run_ica.py

```diff
@@ -74,6 +77,11 @@ def run_ica(subject, session=None):
 
     events, event_id = mne.events_from_annotations(raw)
 
+    if len(np.unique(events[:, 0])) != len(events):
+        warnings.warn('Events are not unique. Uniqifying them')
+        _, idx = np.unique(events[:, 0], return_index=True)
+        events = events[idx]
+
```

Comment:
with https://github.com/mne-tools/mne-python/pull/6688 being merged, we can outsource handling of non-unique events to MNE-Python and remove this code in the study template